package-lock.json

```diff
@@ -2161,11 +2161,6 @@
             "picomatch": "^2.0.5"
           }
         },
-        "mime": {
-          "version": "2.4.6",
-          "resolved": "https://registry.npmjs.org/mime/-/mime-2.4.6.tgz",
-          "integrity": "sha512-RZKhC3EmpBchfTGBVb8fb+RL2cWyw/32lshnsETttkBAyAUXSGHxbEJWWRXc751DrIxG1q04b8QwMbAwkRPpUA=="
-        },
```

Comment:
Hmm I see that these lines exist in `master`, is it possible you accidentally removed these but need to put them back? 🤔

Reply:
Oops. I did `npm install` and there was an npm version issue so It got removed accidently. I have pushed it back.